api/src/models/homepage.model.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * This module exports a "Homepage" mongoose Schema, which represents their homepage.
+ */
+const mongoose = require('mongoose');
+
+const homepageSchema = new mongoose.Schema(
+  {
+    teamId: {
+      type: mongoose.SchemaTypes.ObjectId,
+      ref: 'team',
+      required: true,
+    },
+    aboutUs: [
+      {
+        type: String,
+      },
+    ],
```

Comment:
This would be a great location to document why we're using a string list. Either here or in the file docs above

Reply:
Added comment for string list